### PR TITLE
Format With Prettier 2.0

### PR DESCRIPTION
**Pull request by heathharrelson**
Format code with Prettier 2.0 after update.

### CODE REVIEW

File: js/bucket.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ function trendFn(start, end, interval, overallTarget) {
   const slope = overallTarget / totalIntervals;
 
   // @return {x: String, y: Integer} target point
-  return function(date) {
+  return function (date) {
```
Comment (heathharrelson) [inline js/bucket.js:115]: This is [new default behavior](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot).

File: js/components/ChartForm.vue [hunk 2]
```diff
@@ -567,9 +567,7 @@ export default {
           }
         };
```
Comment (heathharrelson) [inline js/components/ChartForm.vue:569]: The [heuristic used to break call chains was updated](https://prettier.io/blog/2020/03/21/2.0.0.html#improved-method-chain-breaking-heuristic-6685httpsgithubcomprettierprettierpull6685-by-mmkalhttpsgithubcommmkal) to produce less wrapping.

File: test/components/ChartForm_test.js [hunk 3]
```diff
@@ -431,12 +397,7 @@ describe('ChartForm.vue', () => {
     it('creates the expected date field options for the selected event', () => {
       const dateFieldOptions = wrapper.findAll(`${selector.dateFieldSelect} > option`);
       expect(dateFieldOptions.length).toEqual(2); // 1 options plus prompt
-      expect(
-        dateFieldOptions
-          .at(1)
-          .text()
-          .trim()
-      ).toEqual('survey_date (Survey date)'); // sorted by label
+      expect(dateFieldOptions.at(1).text().trim()).toEqual('survey_date (Survey date)'); // sorted by label
```
Comment (lawhead) [inline test/components/ChartForm_test.js:400]: Given that this is all on a single line now it might read better to put the comment above the code.

Comment (heathharrelson) [reply test/components/ChartForm_test.js:400]: Good point. I just glanced over the code after running the formatter, so I'll adjust this and look for anything else that could be improved.

**Pull request merged**